tests/integration/redis/test_basic.php

```diff
@@ -83,6 +86,10 @@ function test_basic() {
   tap_equal(1, $redis->del($key), 'delete key');
   tap_equal(0, $redis->del($key), 'delete missing key');
 
+  tap_assert($redis->mset([$key => 'bar']), 'mset key');
+  tap_equal(['bar'], $redis->mget($key), 'mget key');
```

Comment:
Hi @b-viguier ,

Looks like the tests caught an error in the testcase:
```
Warning: Redis::mget() expects parameter 1 to be array, string given
```

`mget` is expecting an [array](https://github.com/phpredis/phpredis/blob/develop/redis.c#L1275). I think if you put `$key` in brackets `[$key]` it should work.

Reply:
Thanks @zsistla, you're totally right! It's fixed now 👍 

I'm really annoyed, because I did my best to run tests locally and I didn't notice anything wrong 😕 … I'm using the Docker image stored in Github actions: https://github.com/newrelic/newrelic-php-agent/blob/main/.github/actions/ubuntu20-build-action/Dockerfile
The output is very dense, did I miss something? Furthermore, I didn't find where the CI reported the error, are you using tools internal to NR?

Reply:
Thanks  @b-viguier,

No worries.  There are two sets of tests, the [unit tests](https://github.com/newrelic/newrelic-php-agent/tree/main/agent/tests) and our [integration tests](https://github.com/newrelic/newrelic-php-agent/tree/main/tests).

The output from the unit tests can be seen in the github CI output, but we're still in the process of moving from our old CI to github actions for the CI, so the integration tests are are still run internally.